v2/passwordless/quick-setup/frontend.mdx

```diff
@@ -282,8 +286,8 @@ class App extends React.Component {
 </TabItem>
 <TabItem value="vanillajs">
 
-:::caution Not supported
-We do not provide a UI for non ReactJS frameworks yet. So you will have to build your own UI, and integrate them with [our APIs](../apis#frontend-driver-interface).
+:::caution Note
+`supertokens-website` does not provide any UI but can handle session management for you. You can build your own UI and use the [API spec](https://app.swaggerhub.com/apis/supertokens/FDI) to call SuperTokens APIs manually. You can refer to [this example](https://supertokens.com/blog/adding-social-login-to-your-website-with-supertokens) to know how this is done, the example uses social login but the same setup applies to other recipes as well.
```

Comment:
- Mention that you need to use supertokens-website SDK. This providers session management features.
- For login, you need to build your own UI and call the APIs exposed by the backend SDK. You can find the spec here: https://supertokens.io/docs/fdi
- The last sentence is fine, you can make that another point